pkg/controller/kfservice/resources/knative/configuration.go

```diff
@@ -58,25 +65,28 @@ func NewConfigurationBuilder(client client.Client, config *v1.ConfigMap) *Config
 }
 
 func (c *ConfigurationBuilder) CreateKnativeConfiguration(name string, metadata metav1.ObjectMeta, modelSpec *v1alpha1.ModelSpec) (*knservingv1alpha1.Configuration, error) {
-	annotations := make(map[string]string)
+	kfsvcAnnotations := make(map[string]string)
```

Comment:
why rename this variable? We're leaking encapsulation of kfsvc by naming it this way.